demos/ag-alert-backend-lastbyte-resp/README.md

```diff
@@ -0,0 +1,32 @@
+# Setup alert for Backend-Last-Byte-Response time for your Azure Application Gateway 
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/demos/ag-alert-backend-lastbyte-resp/PublicLastTestDate.svg)
+![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/demos/ag-alert-backend-lastbyte-resp/PublicDeployment.svg)
+
+![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/demos/ag-alert-backend-lastbyte-resp/FairfaxLastTestDate.svg)
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/demos/ag-alert-backend-lastbyte-resp/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/demos/ag-alert-backend-lastbyte-resp/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/demos/ag-alert-backend-lastbyte-resp/CredScanResult.svg)
+
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fdemos%2Fag-alert-backend-lastbyte-resp%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fdemos%2Fag-alert-backend-lastbyte-resp%2Fazuredeploy.json)
+
+This template deploys an Azure Monitor alert which notifies you in an event **the average backend last byte response time is greater than [dynamic threshold](https://docs.microsoft.com/en-us/azure/azure-monitor/alerts/alerts-dynamic-thresholds)**.  
+
+**Notes**
+
+- Azure Monitor alert rules are charged based on the type and number of signals it monitors. You may want to visit the [pricing page](https://azure.microsoft.com/en-in/pricing/details/monitor/) before deploying this alert template or can view the estimated cost after the deployment. 
+
+- You will need to create [Azure Monitor Action Group](https://docs.microsoft.com/en-us/azure/azure-monitor/alerts/action-groups) in advance and provide its ResourceID during this deployment. This action group notifies your users when the alert rule is triggered. You can use an existing or create a new one and reuse it for multiple such alerts.
+
+>[!TIP]

```

Comment:
this is not valid markdown for github

Reply:
@bmoore-msft - Can you please specifiy which part exactly is in violation?

Reply:
```[!TIP]``` is for docs.microsoft.com - github renders it literally.

Reply:
Updated the file